README.md

```diff
@@ -25,6 +25,10 @@ API key and building identifier.
 Follow the [Wayfinding guide](https://situm.com/docs/first-steps-for-wayfinding/) if you haven't set
 up a Situm account.
 
+#### Running the example:
+
+Check the [README](./example/README.md) file at the example folder of this repository.
+
```

Comment:
Ao mellor nos como Situm entendemos esto, pero eu especificaría máis:
`#### Try out our example app:`
Check the [`example/README.md`](./example/README.md) guide to create your first React-Native app using Situm SDK
Eu solo pretendo evitarte problemas con QA, dígocho desde a experiencia jejeje